samples/encryption/basic-encryption/sample.md

```diff
@@ -54,10 +54,11 @@ snippet: enableEncryption
 
 This code indicates that encryption should be enabled.
 
-The key is then configured using
+The key is then configured using the `encryptionService` variable. This allows for property encryption using the special `WireEncryptedString` property seen in the `MessageWithSecretData.cs` file.
 
-snippet: ConfigureEncryption
+The following code snippet also shows the optional unobtrusive way to tell NServiceBus which properties to encrypt using the `encryptedPropertyConvention` parameter. In this example, all properties having a name that starts with `Encrypted` will have their value encrypted.
```

Comment:
doesnt this PR add a new sample? if so why add "also shows..." to the existing sample?